app/src/main/res/layout/activity_about.xml

```diff
@@ -62,6 +62,48 @@
                 android:gravity="center"
                 android:text="@string/about_improve" />
 
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:orientation="horizontal"
+                android:layout_marginLeft="@dimen/activity_margin_horizontal"
+                android:layout_marginStart="@dimen/activity_margin_horizontal"
+                android:layout_marginRight="@dimen/activity_margin_horizontal"
+                android:layout_marginTop="@dimen/activity_margin_horizontal"
+                android:layout_marginEnd="@dimen/activity_margin_horizontal"
+                >
+
+                <ImageView
+                    android:layout_width="@dimen/overflow_icon_dimen"
+                    android:layout_height="@dimen/overflow_icon_dimen"
+                    android:src="@drawable/ic_action_globe"
+                    android:contentDescription= "@string/commons_website"
+                    android:layout_margin="@dimen/activity_margin_horizontal"
+                    android:onClick="getOpenWebsiteIntent"
```

Comment:
Binding `onClick` listeners in XML is error prone & easily breaks when refactoring;  **AboutActivity** is a much better place to do this - if you don't want to write the one-line to bind the listener yourself, you could annotate the `getOpenWebsiteIntent()` method with a [Butterknife](http://jakewharton.github.io/butterknife/) "@OnClick" annotation.